runner/buckets.go

```diff
@@ -14,7 +14,10 @@ var (
 
 	customBucketsServer = map[string][]float64{
 		"event_delivery_time": {
-			0.5, 1, 2.5, 5, 10, 30, 60, 300, 600, 1800, 3600, 7200, 10800, 21600, 32400, 86400, // 0.5s, 1s, 2.5s, 5s, 10s, 30s, 1m, 5m, 10m, 30m, 1h, 2h, 3h, 6h, 9h, 24h
+			0.5, 1, 2.5, 5, 10, 30, 60, 120, 180, 300, 600, 1800, 3600, 7200, 10800, 21600, 32400, 86400, // 0.5s, 1s, 2.5s, 5s, 10s, 30s, 1m, 2m, 3m, 5m, 10m, 30m, 1h, 2h, 3h, 6h, 9h, 24h
+		},
+		"event_delivery_from_sent_time": {
+			0.5, 1, 2.5, 5, 10, 30, 60, 120, 180, 300, 600, 1800, 3600, 7200, 10800, 21600, 32400, 86400, // 0.5s, 1s, 2.5s, 5s, 10s, 30s, 1m, 2m, 3m, 5m, 10m, 30m, 1h, 2h, 3h, 6h, 9h, 24h
```

Comment:
do we need all these buckets? can we skip 2.5s, 2m, 3m, 2h as we might not need this much granular information.